Tests/SafeDIToolTests/SafeDIToolCodeGenerationErrorTests.swift

```diff
@@ -40,8 +39,8 @@ final class SafeDIToolCodeGenerationErrorTests {
 
 	// MARK: Error Tests
 
-	@MainActor @Test
-	func run_onCodeWithPropertyWithUnknownFulfilledType_throwsError() async {
+	@Test
+	mutating func run_onCodeWithPropertyWithUnknownFulfilledType_throwsError() async {
```

Comment:
Because we pass in a modifiable `filesToDelete: &filesToDelete` in each `executeSafeDIToolTest` call, we need the test method to be mutating.